object-search/tests/test_objectsearch.py

```diff
@@ -0,0 +1,97 @@
+import base64
+import io
+import json
+import os
+import sys
+import shutil
+import subprocess
+
+import pytest
+
+from jina.flow import Flow
+
+
+num_docs = 3
+top_k = 3
+index_flow_file_path = 'flow-index.yml'
+query_flow_file_path = 'flow-query-original.yml'
+
+
+def config(tmpdir):
+    os.environ['JINA_DATA'] = 'tests/test-data/*.jpg'
+    os.environ['JINA_PORT'] = str(45678)
+    os.environ['PARALLEL'] = str(1)
+    os.environ['SHARDS'] = str(1)
+    os.environ['WORKDIR'] = str(tmpdir)
+
+
+@pytest.fixture
+def setup_env():
+    os.chdir(os.path.join(os.path.dirname(os.path.abspath(__file__)), '..' ))
```

Comment:
Using `os.chdir` creates bugs for the future which are sometimes unforeseeable. I prefer assuming the tests are always run from the root directory `examples`. As far as I can see, this `od.chdir` mainly happens in order to have correct file paths. Ideally, just prefix all file paths with `object-search` and that should do the trick.

Reply:
I mainly had to do this because the `flow` refers to `pods` from the root of example.